kafka_utils/kafka_check/commands/command.py

```diff
@@ -80,6 +90,11 @@ def is_controller(zk, broker_id):
     return broker_id == zk.get_json('/controller').get('brokerid')
 
 
-def is_first_broker(zk, broker_id):
+def get_broker_ids(zk):
+    """Returns a list of integers for broker ids"""
+    return zk.get_brokers().keys()
```

Comment:
Let's surround this with a `list`, to make sure it is compatible for both Python 2 and 3 🙂

Reply:
Thanks. I am fixing it.